client/components/checkout.js

```diff
@@ -31,7 +31,11 @@ class DisconnectedCheckout extends React.Component {
     })
 
     axios.post('/api/orders', {orderItems: orders})
-    axios.delete('/api/cart', {data: {userId: this.props.user.id}})
+    if(this.props.user) {
+        axios.delete('/api/cart', {data: {userId: this.props.user.id}})
+    } else {
+        
```

Comment:
is this supposed to be empty?

Reply:
ooh good catch! i was in the middle of figuring this out, and forgot to get rid of this else case after i realized i didn't need to send any data with the request if i was dealing with a guest